sysinfo.ts

```diff
@@ -64,6 +66,26 @@ export class SysInfo implements ISysInfo {
 		} // if we got an error, assume not working
 	}
 
+	// `android -h` returns exit code 1 on successful invocation. Therefore, we cannot use $childProcess
```

Comment:
Yes, it happens on Mac. I do not know whether Google will make it on other OSes so that I do not make OS check.